pt-BR/mail/chrome/messenger/news.properties

```diff
@@ -36,34 +36,34 @@
 #
 # ***** END LICENSE BLOCK *****
 downloadHeadersTitlePrefix=Receber cabeçalhos
-downloadHeadersInfoText=Há %S novos cabeçalhos de mensagens a receber neste newsgroup.
+downloadHeadersInfoText=Há %S novos cabeçalhos de mensagens a receber neste grupo de notícias.
 htmlNewsErrorTitle=Erro!
 # LOCALIZATION NOTE ( htmlNewsError ): In the following item, translate only "Error!" and "newsgroup server responded:"
-htmlNewsError=<H1>Erro!</H1>Resposta do servidor de newsgroup: 
+htmlNewsError=<H1>Erro!</H1>Resposta do servidor de grupo de notícias:
```

Comment:
CORRECTIONS NEEDED it should be htmlNewsError=<H1>Erro!</H1>Resposta do servidor do grupo de notícias: